docs/sources/developers/plugins/share-a-plugin.md

```diff
@@ -1,120 +1,69 @@
 +++
-title = "Share a plugin"
+title = "Package a plugin"
 type = "docs"
+aliases = ["/docs/grafana/latest/developers/plugins/share-a-plugin/"]
 +++
 
-# Share a plugin
+# Packaging plugins
 
 You've just built your first plugin, and now you want to share it with the world. In this guide, you'll learn how to package and share your plugin with others.
 
 When you build a plugin from source, a `dist` directory is created that contains the production build, or _plugin assets_, for your plugin.
 
-When loading your plugin, Grafana only cares about the plugin assets. Specifically, when the Grafana server starts, it attempts to discover and load plugins like this:
+When loading your plugin, Grafana only cares about the plugin assets. Specifically, when the Grafana server starts, it attempts to discover and load plugins in the following order:
 
 1. Look for a `plugin.json` file in any of the subdirectories in the plugin directory.
 1. If a `plugin.json` was found, try to load the plugin assets from a `dist` directory in the same directory as the `plugin.json` file.
 1. If there's no `dist` directory, try to load the plugin assets from the same directory as the `plugin.json` file.
 
-Now that you know what Grafana needs to load your plugin, let's see how you can share the plugin with other users.
+## Package the plugin
 
-The best way to share your plugin with the world is to publish it on [Grafana Plugins](https://grafana.com/plugins). However, if you're not ready to make your plugin public just yet, you can still share your plugin by hosting the plugin yourself.
+Now that you know what Grafana needs to load your plugin, let's see how you can package and share the plugin with other users.
 
-## Publish your plugin on Grafana.com
-
-To publish a plugin to [Grafana Plugins](https://grafana.com/grafana/plugins), your plugin first needs to be publicly available in a commit on [GitHub](https://github.com).
-
-The commit you submit needs to either:
-
-- Contain a `dist` directory with the plugin assets
-- Contain the plugin assets in the root directory
+### 1. Build the plugin
 
-We strongly recommend that you don't check in the plugin assets to the main branch. Instead, use the following steps to create a release branch that contains the plugin assets.
-
-1. Create a release branch.
+1. Install dependencies.
 
    ```
-   git checkout -b release-0.1.x
+   yarn install --pure-lockfile
    ```
 
-1. Build the plugin assets.
+1. Build plugin assets.
 
    ```
    yarn build
    ```
 
-1. Add the `dist` directory. The `-f` flag adds the directory even if it's ignored by `.gitignore`.
-
-   ```
-   git add -f dist
-   ```
-
-1. Create the release commit.
+1. (Optional) If your data source plugin has a backend plugin, build it as well.
 
    ```
-   git commit -m "Release v0.1.0"
+   mage
```

Comment:
image?

Reply:
[mage](https://github.com/magefile/mage) is the build tool we recommend for building backend plugins.